app/views/users/new.html.erb

```diff
@@ -1,15 +1,15 @@
-<% if flash[:notice] %>
-  <div class="notice"><%= flash[:notice] %></div>
-<% end %>
+<div id="clearance" class="sign-up">
```

Comment:
Hi clearance views! Let's keep only the ones of these we need to modify before merging the PR

(I think that will be only `users/_form`